sample/android/build.gradle

```diff
@@ -11,8 +11,7 @@ buildscript {
         mavenCentral()
     }
     dependencies {
-        // do not upgrade the AGP before https://github.com/react-native-community/upgrade-support/issues/38 gets fixed
```

Comment:
I've read this comment and the issue still looks open.

Nevertheless, the new RN is not compatible with the older AGP.